src/App.vue

```diff
@@ -0,0 +1,467 @@
+<template>
+	<transition v-if="showModal" name="modal">
+		<div id="firstrunwizard" class="modal-mask">
+			<div id="firstrunwizard-navigation">
+				<a v-if="hasPrevious" id="prev" @click="previous">
+					<div class="icon-view-previous icon-white"><span class="hidden-visually">{{ t('firstrunwizard', 'Previous') }}</span></div>
+				</a>
+				<a v-if="hasNext" id="next" @click="next">
+					<div class="icon-view-next icon-white"><span class="hidden-visually">{{ t('firstrunwizard', 'Next') }}</span></div>
+				</a>
+				<a id="close" class="icon-close icon-white" @click="close">
+					<span class="hidden-visually">{{ t('firstrunwizard', 'Close') }}</span>
+				</a>
+			</div>
+			<div class="modal-wrapper" @click.self="close">
+				<div class="modal-container">
+					<div class="modal-header">
+						<div class="firstrunwizard-header">
+							<div class="logo">
+								<p class="hidden-visually">
+									{{ oc_defaults.name }}
+								</p>
+							</div>
+							<h2 v-html="oc_defaults.slogan" />
+							<p />
+						</div>
+					</div>
+					<div class="modal-body">
+						<slot v-if="slides.length > 0" name="body">
+							<transition name="fade" mode="out-in">
+								<div v-if="slides[currentSlide].type === 'inline'" :key="currentSlide" v-html="slides[currentSlide].content" />
+							</transition>
+						</slot>
+					</div>
+					<div class="modal-footer">
+						<button v-if="isLast" class="primary modal-default-button" @click="close">
+							{{ t('firstrunwizard', 'Start using Nextcloud') }}
+						</button>
+					</div>
+				</div>
+			</div>
+		</div>
+	</transition>
+</template>
+<style lang="scss">
+
+	/* Page styling needs to be unscoped, since we load it separately from the server */
```

Comment:
The scoping will not apply since the html from the individual pages is not part of the vue bundle. 

As in this screenshot, if the styles are scoped the .page class will get scoped, but it isn't added to the html element, since that one is inserted as plain html using v-html.

![image](https://user-images.githubusercontent.com/3404133/51207370-e68edb00-190a-11e9-8f90-83aa792dffeb.png)

Reply:
If you really want to scope it, you can still manually scope the full css like we do in the components :)